includes/helper/class-wp-job-manager-site-trust-token.php

```diff
@@ -127,6 +123,7 @@ public function validate( $object_type, $object_id, $token ) {
 		if ( false === $metadatas ) {
 			return false;
 		}
+		$found = false;
```

Comment:
Out of curiosity, any special reason for this change?

Reply:
Ah, now that I saw it in the PR description. Probably related to this, right?

> Fix a rare case where the token might be empty, which could generate weird results

Reply:
Nope, sorry, this change was introduced in 878b90e38f0f6c3c150a2a5d6d34f606389452b6.

The main idea is to allow us to clean the database while NOT stopping the search early. So if for instance for some reason I have the following sequence of metas:

1. token that I want, which is not expired
2. some expired token 2 
3. some expired token 3 
4. some expired token 4 

And it is returned in that exact order...in the previous code, the three expired would never be deleted (well, unless we called `validate` with a token that simply doesn't match anything, or is expired by itself). Now, even if we have a successful result, all the expired tokens will still be deleted as expected.

Hopefully this explains the change well. But if doesn't, please let me know. :)